bitnami/redis-cluster/7.0/debian-11/rootfs/opt/bitnami/scripts/redis-cluster-env.sh

```diff
@@ -94,10 +97,14 @@ export PATH="${REDIS_BIN_DIR}:${BITNAMI_ROOT_DIR}/common/bin:${PATH}"
 export REDIS_DAEMON_USER="redis"
 export REDIS_DAEMON_GROUP="redis"
 
+
 # Redis settings
+# Added REDIS_RDB_ENABLED and REDIS_RDB_POLICY variables
 export REDIS_DISABLE_COMMANDS="${REDIS_DISABLE_COMMANDS:-}"
 export REDIS_DATABASE="${REDIS_DATABASE:-redis}"
 export REDIS_AOF_ENABLED="${REDIS_AOF_ENABLED:-yes}"
+export REDIS_RDB_ENABLED="${REDIS_RDB_ENABLED:-no}"
+export REDIS_RDB_POLICY="${REDIS_RDB_POLICY:-900#1 300#10 60#10000}"
```

Comment:
By default we can leave the variable empty, similar to `save ""`
```suggestion
export REDIS_RDB_POLICY="${REDIS_RDB_POLICY:-}" # eg. 900#1 300#10 60#10000
```